api/src/pcapi/core/offerers/models.py

```diff
@@ -218,6 +218,8 @@ class Venue(PcObject, Model, HasThumbMixin, ProvidableMixin, NeedsValidationMixi
         "Criterion", backref=db.backref("venue_criteria", lazy="dynamic"), secondary="venue_criterion"
     )
 
+    dms_token = Column(Text, nullable=True, unique=True)
```

Comment:
Je me demande si on gagne vraiment à mettre cette colonne _nullable_. Si je comprends bien, seuls les lieux numériques n'ont pas de token. Mais ça n'est pas un besoin, non ? Par conséquent, j'ai l'impression que ce serait plus simple de toujours avoir un token (qui ne serait pas utilisé pour les lieux numériques) et donc un champ _**not** nullable_, qui garantirait mieux la cohérence des données. Qu'en penses-tu ?

Reply:
Je pensais rendre la colonne non-nullable après que tous les lieux auront été rattrapés.